### DISCUSSION TITLE
What is this useful for? Example use cases?

### DISCUSSION THREAD

**Discussion by BevanR** [vercel/satori#261] [Q&A]
What higher-level problems is Satori useful for? Why would one want to convert HTML to SVG instead of just craft SVG? What are some examples of how this is already being used? What are the use cases it is intended for?

**Comment by styfle** [comment]
Take a look at the announcement blog post here https://vercel.com/blog/introducing-vercel-og-image-generation-fast-dynamic-social-card-images

**Comment by wei** [comment]
One use-case is my project [socialify](https://github.com/wei/socialify) which generates social images for any github project to be included inside README, websites, and beyond. 

Satori saves us the headache of manually laying out components inside svgs but instead, we just need to focus on laying out designs using html/css and converting it to svg/png easily.

[Link][socialify-edit-link] / Live Preview:
![socialify](https://socialify.git.ci/wei/socialify/image?description=1&font=Raleway&issues=1&language=1&pattern=Charlie%20Brown&stargazers=1&theme=Auto)

[socialify-image]: https://socialify.git.ci/wei/socialify/image?description=1&font=Raleway&issues=1&language=1&pattern=Charlie%20Brown&stargazers=1&theme=Auto
[socialify-edit-link]: https://socialify.git.ci/wei/socialify?description=1&font=Raleway&issues=1&language=1&pattern=Charlie%20Brown&stargazers=1&theme=Auto
